CEGameApp.API/Migrations/20200227153038_extendedUsersClass.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace CEGameApp.API.Migrations
+{
+    public partial class extendedUsersClass : Migration
```

Comment:
going to guess this was auto generated because this class name is not great lol

Reply:
yes it was auto generated.